src/wmnds/components/content-tiles/small2/small2.njk

```diff
@@ -0,0 +1,14 @@
+{% macro wmndsContentTilesSmall2(params) %}
+<div class="wmnds-content-tile">
+   <div class="wmnds-content-tile__title">
+      <h2>Hagley Road and Five Ways area</h2>
+   </div>
+   <div class="wmnds-content-tile__copy">
+      <ul class="wmnds-content-tile__copy-list">
+         <li><span>Location:</span> Birmingham city centre</li>
```

Comment:
Not sure why spans are here? If you were using these to pinpoint styles to, then that isn't BEM standard.